engine/src/main/java/org/terasology/rendering/cameras/PerspectiveCamera.java

```diff
@@ -49,8 +51,9 @@
 
     private Vector3f tempRightVector = new Vector3f();
 
-    public PerspectiveCamera(PerspectiveCameraSettings cameraSettings) {
-        this.cameraSettings = cameraSettings;
+    public PerspectiveCamera(WorldProvider worldProvider, RenderingConfig renderingConfig) {
+    	super(worldProvider, renderingConfig);
+        this.cameraSettings = renderingConfig.getCameraSettings();
```

Comment:
I'm not sure why, but that extra space is only showing in the commit changes, if you go into the actual file it doesn't have that extra space